Source/WTF/wtf/PlatformHave.h

```diff
@@ -1623,6 +1623,12 @@
 #define HAVE_AUDIOFORMATPROPERTY_VARIABLEPACKET_SUPPORTED 1
 #endif
 
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000) \
+    || (PLATFORM(MACCATALYST) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000) \
+    || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000)
+#define HAVE_UPDATED_CORE_CRYPTO_SIGNATURES 1
```

Comment:
Maybe we should use a slightly more descriptive name than "updated". Is there a CoreCrypto version number or scope this `#define` to curve/ed25519 signatures?

Reply:
`_WITH_INT_RETURN_VALUE`?

Reply:
CoreCrypto does have version numbers(just confirmed).
`_WITH_INT_RETURN_VALUE` seems better.. I can also add the commit hash of CC for which we are adding this ifdef.
How about `UPDATED_CORE_CRYPTO_SIGNATURES_WITH_INT_RETURN_VALUES_d0f475e405` ?

Reply:
I don't see the reason for adding commit hash.